src/store/metadata/stub/content/editions/schools-and-nursery-schools-rated_2023-05.json

```diff
@@ -0,0 +1,118 @@
+{
+    "@context": "https://staging.idpd.uk/#ns",
+    "@id": "https://staging.idpd.uk/datasets/schools-and-nursery-schools-rated/editions",
+    "@type": "hydra:Collection",
+    "title": "Schools and nursery schools rated good or outstanding",
+    "editions": [
+      {
+        "@id": "https://staging.idpd.uk/datasets/schools-and-nursery-schools-rated/editions/2023-05",
+        "@type": "dcat:Dataset",
+        "in_series": "https://staging.idpd.uk/datasets/schools-and-nursery-schools-rated",
+        "identifier": "2023-05",
+        "title": "Schools and nursery schools rated good or outstanding",
+        "summary": "Percentage of state-funded provided for schools that are rated",
```

Comment:
Maybe change this to 'Percentage of state-funded schools that are rated good or outstanding'

Reply:
noted